src/index.html

```diff
@@ -1,16 +1,54 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="http://localhost:8080/"
+      method="get"
+      class="form"
+      >
+      <img
+        src="images\Search.svg"
+        alt="Search bar"
+        class="svg"
+      >
+      <input
+      type="text"
+      data-qa="big"
+      class="bar"
+      placeholder="Try &quot;Los Angeles&quot;"
+      name="search-small-bar"
+      >
```

Comment:
```suggestion
      <input
        type="text"
        data-qa="big"
        class="bar"
        placeholder="Try &quot;Los Angeles&quot;"
        name="search-small-bar"
      >
```
check your [code style](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) 🥲

Reply:
(all over the code 😉)

Reply:
Все ще невиправлений коментар